packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -386,7 +411,7 @@ const SelectedEntriesModalContent = ({
         }
         endActions={
           <Flex gap={2}>
-            <Button onClick={() => refetchModalData()} variant="tertiary" loading={isFetching}>
+            <Button onClick={refetchList} variant="tertiary" loading={isFetching}>
```

Comment:
`refetch` is broken as you are invalidating list view not the modal content

Reply:
Do you say is broken for something you see? 🤔  Modal content is the same that listView content, just filtered to show only selected entries.

In my opinion the last behaviour of fetch entries by separated doesn't have sense because I don't want to make the same request twice, just use the information that I already have